DataSetExplorer/AnnotationConsistency/IAnnotationConsistencyChecker.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace DataSetExplorer
+{
+    interface IAnnotationConsistencyChecker
+    {
+        public void CheckMetricsSignificanceBetweenAnnotatorsForSeverity(int severity);
```

Comment:
Result<string> or event Result<more-complex-object>.

This will be especially important for the Web, but we should put everything on good footing for the MVP so that we may easily swipe out the Console UI.